v2.6/getting-started/kubernetes/installation/hosted/canal/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Canal Hosted Install
+canonical_url: 'https://docs.projectcalico.org/v3.0/getting-started/kubernetes/installation/hosted/kubeadm/'
+---
+
+## About Canal
+
+Canal aims to allow users to easily deploy Calico and flannel networking
+together as a unified networking solution - combining Calico’s
+industry-leading network policy enforcement with the flannel overlay and
+non-overlay network connectivity options.
+
+Note that the Canal currently uses the Calico and flannel projects as is with
+no code modifications to either. Canal is simply a deployment pattern
+for installing and configuring the projects to work together seamlessly as
+single network solution from the point of view of the user and orchestration
+system.
+
+
+## Installation
+
+If you have already built your cluster, please review the
```

Comment:
I think this reads a bit odd - mostly the "It is likely you'll need to recreate your cluster" bit.

What about something more like:

"When deploying your kubernetes cluster, please make sure it meets the following requirements. An easy to way to create a cluster which meets these requirements is by following the official kubeadm guide"